libs/feature/record/src/lib/links/link-helper.service.ts

```diff
@@ -11,6 +11,24 @@ export class LinkHelperService {
   hasLinks(record: MetadataRecord): boolean {
     return 'links' in record
   }
+  hasDownloadProtocols(protocols: string[]): boolean {
+    return protocols
+      .map((protocol) => {
+        return { protocol }
+      })
+      .some((link: Partial<MetadataLink>) =>
+        this.isDownloadLink(link as MetadataLink)
+      )
+  }
+  hasMapApiProtocols(protocols: string[]): boolean {
+    return protocols
+      .map((protocol) => {
+        return { protocol }
+      })
+      .some((link: Partial<MetadataLink>) =>
+        this.isMapApiLink(link as MetadataLink)
+      )
+  }
```

Comment:
The classifier classifies a link to binds it to usages. The helper gives helper function to quickly get informations about a record, a link or something else. To me it make sense, but of couse we could fusion them together.
I would like to keep the helper functions though.

Reply:
Hmm, ok, I understand that the helper functions make it easier to use the classification logic. I still think that this service isn't adding much value, and is creating confusion since the same directory contains 3 services (`link-classifier`, `link-utils` and `link-helpers`). Maybe merging `link-utils` and `link-helpers` would be the better option. Anyway, this can be done later on, no issue!